### PR TITLE
collection: add individual researcher message

### PR DESCRIPTION
**Pull request by diegoviola**

### CODE REVIEW

File: app/views/collection/new.html.slim [hunk 1]
```diff
@@ -1,5 +1,9 @@
 .litebox-embed(style="width:550px")
   h1 Create New Collection
+
+  - if current_user.account_type == "Individual Researcher"
+    p(style="color: #FF0000") Individual Researcher Accounts are limited to a single collection. To upgrade your account, contact support@fromthepage.com.
+
```
Comment (saracarl) [inline app/views/collection/new.html.slim]: So this should not show the form for creating a new collection if 
current_user.account_type == "Individual Researcher"
AND
current_user collection count => 1  

It should only show the message if 
current_user.account_type == "Individual Researcher"
AND
current_user collection count => 1 

In other words, one or the other.

**Comment by saracarl** [comment]
Tests passed locally, though!

**Comment by benwbrum** [comment]
We still need to hide the form if we display the warning message.

**Comment by diegoviola** [comment]
@benwbrum done.

Note that I had to change the account_type to "Trial" for those specific tests, otherwise the form gets disabled and causes the tests to break.

**Pull request merged**